Makefile

```diff
@@ -39,14 +38,11 @@ check_assets: assets
 		exit 1; \
 	fi
 
-build_tsdb:
-	GO111MODULE=$(GO111MODULE) $(GO) build -o $(TSDB_BIN) $(TSDB_CLI_DIR)
-
-bench_tsdb: build_tsdb
+bench_tsdb:
```

Comment:
I think we should keep the build promu target

```
build_promu:
	GO111MODULE=$(GO111MODULE) $(GO) build ./cmd/promtool	

bench_tsdb: build_promu
.
.
.
```

Reply:
Done